packages/neo-one-client/src/user/keystore/LedgerHandler.ts

```diff
@@ -0,0 +1,148 @@
+import { crypto } from '@neo-one/client-core';
+import { HWLedger, LedgerTransport } from '@neo-one/client-switch';
+
+export interface LedgerResponse {
+  readonly message: Buffer;
+  readonly status: Buffer;
+}
+
+export interface LedgerQueueRequest {
+  readonly params: string;
+  readonly msg: string;
+  readonly statusList: ReadonlyArray<number>;
+  readonly resolve: (value: LedgerResponse) => void;
+  // tslint:disable-next-line:no-any
+  readonly reject: (error: Error) => void;
+}
+const VALID_STATUS = 0x9000;
+// const MSG_TOO_BIG = 0x6d08;
+// const APP_CLOSED = 0x6e00;
+// const TX_DENIED = 0x6985;
+// const TX_PARSE_ERR = 0x6d07;
+
+const BIP44 = (accountIn = 0) => {
+  const account = accountIn.toString(16);
+
+  return `8000002C800003788000000000000000${'0'.repeat(8 - account.length)}${account}`;
+};
+
+const getParams = (params: string) => {
```

Comment:
This is silly, just make the `send` method below well typed for exactly what they should be - don't split strings and do regex matches and other nonsense.